docs/src/modules/components/DiamondSponsors.js

```diff
@@ -33,17 +33,26 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+// Persisted for the whole session
+const random = Math.random();
```

Comment:
Why do we not want to cycle through the diamond sponsors while browsing?

Reply:
The logic avoids layout jump for a given session.

Reply:
Could you include this in the comment? You're talking about re-mounting in the navbar which can be fixed. Please be specific since we don't do this for Ads.

Reply:
I have added this in the comment. Also, we will increase the display split to 25% starting next month (up from 5%).

Reply:
Layout jumps where? The question you're trying to answer is why this doesn't apply to `Ad`

Reply:
The logo doesn't reserve any space. When we decide to display it, we move the menu items down by 25px. The objective is to avoid this jump down and up
I don't see the link with the Ad component. There, we always display an Ad. We have a few random usage for increasing the cardinality of ad people see.

Reply:
>  There, we always display an Ad. 

But we cycle through them so it's not immediately clear they don't cause a layout shift as well. 

I thought this variable decides which diamond sponsor we display not that we display one. 

Maybe add a descriptive name for the variable so that it's obvious what this is used for? In the current form it can be used for anything "random" which is most the time the wrong semantics.

Reply:
Yeah ok, changing the name